dev_guide/builds.adoc

```diff
@@ -105,6 +105,39 @@ The following options are specific to the
 link:../architecture/core_concepts/builds_and_image_streams.html#source-build[S2I
 build strategy].
 
+[[force-pull]]
+
+=== Force Pull
+
+By default, if the builder Docker image specified in the  build confirguration is available in OpenShift's local,
```

Comment:
s/builder Docker image/builder image/
s/confirguration/configuration/
s/in OpenShift's local, internal registry/in the OpenShift internal registry/